src/v3/src/components/Widget/index.tsx

```diff
@@ -134,8 +135,13 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   const languageCode = getLanguageCode(widgetProps);
   const languageDirection = getLanguageDirection(languageCode);
   const { stateHandle, unsetStateHandle } = useStateHandle(widgetProps);
+  // Odyssey language codes use '_' instead of '-' (e.g. zh-CN -> zh_CN)
+  const odyLanguageCode: string = languageCode.replace('-', '_');
+  // Only pick Odyssey translation keys from translation bundle
+  const odyTranslationOverrides: TranslationOverrides<string> = {
+    [odyLanguageCode]: pick(Bundles.login, odysseyI18nResourceKeysList)
+  } as TranslationOverrides<string>;
```

Comment:
Was debating whether these should be moved to a utility file, @glenfannin-okta do you have any ideas on whether these would fit better in a different file?

Also, what's our stance on `lodash` and `underscore`? I could implement pick natively if needed, see: https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore?tab=readme-ov-file#_pick

Reply:
Left a comment on the `odyTranslationOverrides`, but re: the util, I usually only lean into creating a util if it is going to be re-used or if there is enough logic to warrant a separate dedicated file. This seems pretty straightforward and may not warrant a separate util file.

Re: lodash, initially we were using it alot, then we began to only use when absolutely necessary due to the size of lodash and us not having tree shaking. I think in this case, if there is simple logic that can be created to avoid using lodash functions we should go for that instead.